src/Dev/FixFilePermissionsTask.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SilverStripe\Dev\Tasks;
+
+use SilverStripe\Assets\File;
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\ORM\Queries\SQLUpdate;
+use SilverStripe\Security\InheritedPermissionFlusher;
+
+/**
+ * Files imported from SS3 might end up with broken permissions if there is a case conflict.
+ * This task resets the `CanViewType` of files that are `NULL`.
+ * You need to flush your cache after running this task via CLI.
+ */
+class FixFilePermissionsTask extends BuildTask
+{
+    private static $segment = 'FixFilePermissionsTask';
+
+    protected $title = 'Fix File Permissions';
+
+    protected $description = '
+    Reset the permission of files that have a `NULL` CanViewType value.
+    You need to flush your cache after running this task via CLI.
+    ';
+
+    public function run($request)
```

Comment:
Should have some logging - e.g. `Repaired X folders with broken CanViewType settings`. Running two raw SQL queries (one count, one update) is fine for that